### ISSUE TITLE
add USGS and update Google analytics scripts

### ISSUE THREAD

**Issue by wdwatkins** [USGS-VIZLAB/vizlab#137]

**Comment by ldecicco-USGS** [comment]
https://github.com/USGS-VIZLAB/vizlab/blob/master/inst/templates/fullPage.mustache#L31

should be:
`
ga('create', '{{ . }}', 'usgs.gov');
```
I think?

**Comment by ldecicco-USGS** [comment]
And just need to add:
```
<script type="application/javascript" src="https://www2.usgs.gov/scripts/analytics/usgs-analytics.js"></script>
```
somewhere in there too

**Comment by aappling-usgs** [comment]
@wdwatkins - is this issue obsolete or still a good idea?

**Comment by ldecicco-USGS** [comment]
Be sure to check with Jim too, at one point we were missing the USGS and DOI analytics. I'm not sure they are mandatory, but when things go viral, they would be noticed even more if we had the right scripts set up.

**Comment by wdwatkins** [comment]
I'm not sure on Laura's first comment, but that script seems to indicate it should be used.  https://www2.usgs.gov/scripts/analytics/usgs-analytics.js

**Comment by aappling-usgs** [comment]
jim says, "yeah, that's actually essential, required by usgs, doi, and omb policy" - we will prioritize this.

**Comment by wdwatkins** [comment]
On semi-related topic, there is a new implementation of Google's analytics script that we might want to start using for our own analytics.  It is now the default option that google tells you to use when you make a new analytics ID.

**Comment by aappling-usgs** [comment]
Keeping up sounds good. Are we talking a 1-2 line edit for that, too? Do you have a link you could share here?

**Comment by wdwatkins** [comment]
https://developers.google.com/analytics/devguides/collection/gtagjs/migration

**Comment by aappling-usgs** [comment]
there are also analytics buried within the javascript functions attached to the VIZLAB javascript namespace - those will need updating, too

**Comment by jread-usgs** [comment]
the things I was talking about include `vizlab` scoped GA functions. So, from `maria`:
![image](https://user-images.githubusercontent.com/2349007/31462974-15f13bc8-ae94-11e7-81af-0f7c8fc259f5.png)

**Comment by jread-usgs** [comment]
those functions are in `/inst/js/vizlab.js`

**Comment by ldecicco-USGS** [comment]
- [ ] usgs-analytics.js is already in the mustache
- [ ] the line `ga('create', '{{ . }}', 'auto');` is in our script, but seems to work?
- [ ] investigate "new implementation of Google's analytics script"
- [ ] investigate "vizlab scoped GA functions"

**Comment by aappling-usgs** [comment]
closed by #299
